src/sass/styles.scss

```diff
@@ -432,6 +432,10 @@ span[role="link"] {
   }
 }
 
+.formio-choices .usa-select {
+  background-image: none;
+}
```

Comment:
can you also add border-width:"1px" and border-color = "#565c65"; to match with usa-input.The look and feel will be similar with input when renders with USWDS viewer.

Reply:
"usa-select" class already has these properties, so it will look the same as input